app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -27,11 +27,12 @@ const MapImpl: React.FunctionComponent = () => {
   const state: MapState = useSelector(
     (rootState: RootState) => rootState.app.map
   );
+  //console.log(state);
 
   // Load features on first render
   const featuresQueryResult = useQuery<FeaturesQuery, FeaturesQueryVariables>(
```

Comment:
I'd like to see this called something like initialFeaturesQueryResult, with appropriate GraphQL query names like InitialMapFeaturesQuery (or MapInitialFeaturesQuery). We may be requesting features from other components, in which case "FeaturesQuery" will be too generic a name.